client/src/styles/App.css

```diff
@@ -22,8 +22,12 @@
   width: 80px;
 }
 
+.btn-primary:hover,
+.btn-primary:active,
+.btn-primary:visited,
+.btn-primary:focus,
```

Comment:
Have you tried overriding :focus with inline style on the button?

Reply:
I'll try it on another branch